src/WorkflowGui/Resources/translations/admin.en.yml

```diff
@@ -114,3 +114,4 @@ workflow_gui: 'Workflow GUI'
 workflow_problem_creating_workflow_invalid_characters: 'Please use only these characters: "a-z", "A-Z" and "_"'
 workflow_gui_workflow_with_name_already_exists: 'Workflow with bane already exists'
 workflow_gui_clone: 'Clone'
+workflow_gui_enable_message: 'Please enable and save the workflow, before you can see the graph'
```

Comment:
This sentence is structurally incorrect. Suggestion:
Please enable and save the workflow to display graph visualization.

or something similar.

Reply:
Ah my bad English again, thank you

Reply:
I don't see an update yet.

Reply:
@kjkooistra-youwe should be oke now